wkcuber/wkcuber/export_wkw_as_nifti.py

```diff
@@ -0,0 +1,206 @@
+import logging
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import nibabel as nib
+import numpy as np
+
+from wkcuber.api.bounding_box import BoundingBox
+from wkcuber.api.dataset import Dataset, LayerCategories
+from wkcuber.mag import Mag
+from wkcuber.utils import (
+    add_distribution_flags,
+    add_verbose_flag,
+    parse_bounding_box,
+    setup_logging,
+)
+
+
+def create_parser() -> ArgumentParser:
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--source_path", "-s", help="Directory containing the wkw file.", required=True
+    )
+
+    parser.add_argument(
+        "--destination_path",
+        "-d",
+        help="Output directory for the generated nifti files.",
```

Comment:
```suggestion
        help="Output directory for the generated nifti file.",
```

since the script only outputs one nifti file, right?

Reply:
No, it actually writes one file per layer. I updated the description to reflect that.